doc/configuration.rst

```diff
@@ -37,6 +37,7 @@ This is the preferred configuration source, and may gain features not available
    required, one of:
 
    - `toml <https://pypi.org/project/toml/>`__
+   - `tomli <https://pypi.org/project/tomli/>`__
```

Comment:
might also make sense to drop a link to https://peps.python.org/pep-0680/

Reply:
Yeah, rephrased the whole thing since it's in stdlib now, after all ;-).